Makefile

```diff
@@ -6,6 +6,7 @@ build:
 
 test: build
 	go test -v -count=1 ./testing
+	go test -v -count=1 ./validation
```

Comment:
Can you make this step invoke testing for all tests excluding `./testing`?

This way, if someone adds a new unit test in the future, they won't have to go through this exercise again.

Additionally, you can remove the `-count=1` option as it's only relevant for system tests.

Also, could you please separate this in a new step - e.g. `unit_test`. This way, if someone sees his CI step failing, he will know whether it's from the unit tests or integration tests.

Finally, could you modify `.github/workflows/master.yaml` to use this new make step to invoke all unit tests in CI?